index.html

```diff
@@ -10,6 +10,7 @@
     <title>CV-Marcin Laszczynski</title>
 </head>
 <body>
+<div class="wrapper">
```

Comment:
The indentation is wrong here. Since `wrapper` is inside `<body>`, it should be indented more to the right.